src/styles/_contributors.scss

```diff
@@ -100,13 +100,13 @@ div.inPermalink {
     text-align: center;
     vertical-align: top;
     display: inline-block;
-    width: (100% / 5);
-    height: (100% / 5);
+    width: math.div(100%, 5);
```

Comment:
Would it be better to change this to 20% and the one below to 33.3% since both sides of the equations are constants?